docs/modules/ROOT/pages/references/quality-requirements/usability/mearuring-scope-for-invoicing.adoc

```diff
@@ -0,0 +1,24 @@
+= Scope metrics are colected for invoicing 
+
+Source::
+Invoicing adapter
+
+Stimulus::
+Calculate invoice
+
+Environment::
+{global}
+
+Artifact::
+Metric storage for invoicing
+
+Response::
+A time series of usage metrics
+
+Response measure::
+A time series is returned per pod.
```

Comment:
Maybe mention that this is also per namespace and per zone?
Actually, it might be a good idea to further specify the minimum data fields?
```suggestion
Response measure::
A time series is returned per pod.
The series contains in the minimum the namespace name, {zone} identifier and total used memory in bytes.
```